examples/with-sentry/README.md

```diff
@@ -0,0 +1,31 @@
+[![Deploy to now](https://deploy.now.sh/static/button.svg)](https://deploy.now.sh/?repo=https://github.com/zeit/next.js/tree/master/examples/with-sentry)
+
+# Sentry example
+
+## How to use
+
+Download the example [or clone the repo](https://github.com/zeit/next.js):
+
+Install it and run:
+
+**npm**
+```bash
+npm install
+npm run dev
+```
+
+**yarn**
+```bash
+npm install
+npm run dev
+```
+
+Deploy it to the cloud with [now](https://zeit.co/now) ([download](https://zeit.co/download))
+
+```bash
+now
+```
+
+## About example
+
+This example show you how to add Sentry to catch errors in next.js
```

Comment:
Try to add info on how to get the Sentry DSN (at least a link)

Reply:
Thanks for the feedback, will fix these things